src/serlo-editor/plugins/page-layout/editor.tsx

```diff
@@ -40,6 +50,38 @@ export const PageLayoutEditor: React.FunctionComponent<
     </>
   )
 
+  function renderPluginToolbar() {
+    if (!focused) return null
+
+    const layoutStrings = editorStrings.plugins.layout
```

Comment:
same as above, should be `const pageLayoutStrings = useEditorStrings().plugins.pageLayout`